helm/cosmo/charts/studio/templates/deployment.yaml

```diff
@@ -60,6 +60,11 @@ spec:
                 configMapKeyRef:
                   name: {{ include "studio.fullname" . }}
                   key: slackClientId
+            - name: CUSTOM_HEAD_SCRIPTS
+              valueFrom
+                configMapKeyRef:
+                  name: {{ include "studio.fullname" . }}
+                  key: customHeadScripts
```

Comment:
This reference to the config map. You need to a mapping here https://github.com/wundergraph/cosmo/blob/fix/custom-scripts/helm/cosmo/charts/studio/templates/config-map.yaml